kiwi/bootloader/template/grub2.py

```diff
@@ -291,6 +295,7 @@ def get_disk_template(
             template_data += self.menu_entry
             if failsafe:
                 template_data += self.menu_entry_failsafe
+        template_data += self.menu_entry_boot_snapshots
```

Comment:
When somebody is doing `+=` on Python strings and it is over templates, then I am like:
[![Please do not](https://img.youtube.com/vi/JncgoPKklVE/0.jpg)](https://www.youtube.com/watch?v=JncgoPKklVE)
https://www.youtube.com/watch?v=JncgoPKklVE